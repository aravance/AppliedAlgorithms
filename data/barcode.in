9
01
123
1234
54321
95551212
6
66
666
78
1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890
0
