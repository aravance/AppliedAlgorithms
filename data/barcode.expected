NnNnNnNwWnWwNnWnN
NnNnNwNnWnWnNwWnN
NnNnNwNnWnWnNwNwWwNnNnWnWnN
NnNnWnNwNnNnWwWnWnNwNnNwWnN
NnNnNwNnWwWnNnNwNwWnNnWnNwWnNnNnWwWnN
NnNnNwWnNwWnNnWwNnWwNnNnWnNwNnNnWwWnNwNnNnWwWnN
NnNnNnNwWwWnNnWnN
NnNnNnWwWwNnNnWnN
NnNnNnNwWwWnNnNnWwWwNnNnWnN
NnNnNwNnNnWwWnWnN
NnNnWnNwNnNnWwWnWnNwNnNwWnNwWwNnNnNwNnNnWwWnNnWnNwWwNnWnNwNnNnWwWnWnNwNnNwWnNwWwNnNnNwNnNnWwWnNnWnNwWwNnWnNwNnNnWwWnWnNwNnNwWnNwWwNnNnNwNnNnWwWnNnWnNwWwNnWnNwNnNnWwWnWnNwNnNwWnNwWwNnNnNwNnNnWwWnNnWnNwWwNnWnNwNnNnWwWnWnNwNnNwWnNwWwNnNnNwNnNnWwWnNnWnNwWwNnWnNwNnNnWwWnWnNwNnNwWnNwWwNnNnNwNnNnWwWnNnWnNwWwNnWnNwNnNnWwWnWnNwNnNwWnNwWwNnNnNwNnNnWwWnNnWnNwWwNnWnNwNnNnWwWnWnNwNnNwWnNwWwNnNnNwNnNnWwWnNnWnNwWwNnWnNwNnNnWwWnWnNwNnNwWnNwWwNnNnNwNnNnWwWnNnWnNwWwNnWnNwNnNnWwWnWnNwNnNwWnNwWwNnNnNwNnNnWwWnNnWnNwWwNnWnN
